client/coral-admin/src/containers/Community/Community.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import I18n from 'coral-framework/i18n/i18n';
+import translations from '../../translations';
+import {Grid, Cell} from 'react-mdl';
+
+import styles from './Community.css';
+import Table from './Table';
+import Loading from './Loading';
+import NoResults from './NoResults';
+import Pager from './Pager';
+
+const Community = ({searchValue, onKeyDownHandler, onChangeHandler, commenters, isFetching, onHeaderClickHandler, page, totalPages, onNewPageHandler}) => (
```

Comment:
this is where we shoud just have the param named `props` and do `props.property` to avoid this huge param extraction